sklearn/model_selection/_validation.py

```diff
@@ -449,6 +450,13 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
     return_parameters : bool, default=False
         Return parameters that has been used for the estimator.
 
+    split_progress : list or tuple, optional, default: None
+        A list or tuple of format (<current_split_id>, <total_num_of_splits>)
```

Comment:
iterator of tuples?

Reply:
You mean that `split_progress` comes with an iterator built on top of it, right? But the type of `split_progress` when it is passed to `_fit_and_score` is really "just" a `<class 'tuple'>`, if I understand correctly. I'd rather keep the description in the docstring as simple as possible... WDYT?

Reply:
my bad, I thought this was a list of tuples or a tuple of tuples.

Then just `tuple` is fine, but the current version is good too. resolving!